src/biharkrishi/fetch-db-response/fetchdbresponse.service.ts

```
@@ -0,0 +1,65 @@
+      select: { 
+        question: true, 
+        response: true, 
+        intent: true,         // Include intent
+        schemeId: true,       // Include schemeId
+        scheme: {             // Include related scheme name
+          select: { name: true }
+        }
+      },
+    });
+
+    if (mainQuestion && mainQuestion.scheme) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)